packages/web-components/src/divider/divider.styles.ts

```diff
@@ -0,0 +1,29 @@
+import { css } from '@microsoft/fast-element';
+
+import {} from '../theme/design-tokens.js';
+
+export const styles = css`
+  :host {
+  }
```

Comment:
We'll want an initial display on the component and to set any defaults in CSS only to keep a clean DOM (not sprouting/reflecting attributes if we don't need to)